heron/config/src/yaml/conf/local/healthmgr.yaml

```diff
@@ -10,10 +10,16 @@ heron.healthmgr.metricsource.url: http://localhost:8888
 ## list of policies to be executed for self regulation
 #heron.class.health.policies:
 #  - dynamic-resource-allocation
+#  - auto-restart-backpressure-container
 #
 ## configuration specific to individual policies listed above
 #dynamic-resource-allocation:
 #  health.policy.class: com.twitter.heron.healthmgr.policy.DynamicResourceAllocationPolicy
 #  health.policy.interval.ms: 120000
 #  BackPressureDetector.noiseFilterMillis: 20
 #  GrowingWaitQueueDetector.limit: 5
+#auto-restart-backpressure-container:
+#  health.policy.class: com.twitter.heron.healthmgr.policy.AutoRestartBackpressureContainerPolicy
+#  health.policy.interval.ms: 120000
+#  BackPressureDetector.noiseFilterMillis: 20
+#  GrowingWaitQueueDetector.limit: 5
```

Comment:
The `GrowingWaitQueueDetector` is not used by this policy. This config can be removed.

Reply:
removed